autonomy/cli/deploy.py

```diff
@@ -113,77 +120,125 @@ def build_group() -> None:
     default=False,
     help="Remove existing build and overwrite with new one.",
 )
+@click.option(
+    "--skip-images",
+    is_flag=True,
+    default=False,
+    help="Specify whether to build images or not.",
+)
+@registry_flag()
 @password_option(confirmation_prompt=True)
-def build_deployment(  # pylint: disable=too-many-arguments
+def build_deployment(  # pylint: disable=too-many-arguments, too-many-locals
     service_id: PublicId,
     keys_file: Path,
     deployment_type: str,
     output_dir: Path,
     packages_dir: Path,
     dev_mode: bool,
     force_overwrite: bool,
+    registry: str,
     number_of_agents: Optional[int] = None,
     password: Optional[str] = None,
     version: Optional[str] = None,
+    skip_images: bool = False,
 ) -> None:
     """Build deployment setup for n agents."""
 
-    packages_dir = Path(packages_dir)
-    keys_file = Path(keys_file)
-    build_dir = Path(output_dir, "abci_build")
-
-    if not packages_dir.is_dir():
-        raise click.ClickException(
-            f"Packages directory does not exists @ {packages_dir}"
-        )
+    packages_dir = Path(packages_dir).absolute()
+    keys_file = Path(keys_file).absolute()
+    build_dir = Path(output_dir, "abci_build").absolute()
 
     if build_dir.is_dir():
         if not force_overwrite:
             raise click.ClickException(f"Build already exists @ {output_dir}")
         shutil.rmtree(build_dir)
```

Comment:
this gives issues on windows right? I will rename and place this into a helper function instead of conftest.py on open-aea and add this as a note to a ticket on finalizers for clean-up (https://trello.com/c/PKbOXtsU/883-finalizers)

https://github.com/valory-xyz/open-aea/blob/a6eab0e13465383888dd52aa04b3780c3dbdfd59/tests/conftest.py#L420-L429